src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -33,6 +34,18 @@ public function getConfigTreeBuilder(): TreeBuilder
 
         $rootNode
             ->docUrl('https://symfony.com/doc/{version:major}.{version:minor}/reference/configuration/twig.html', 'symfony/twig-bundle')
+            ->beforeNormalization()
+            ->ifTrue(fn ($v) => \is_array($v) && \array_key_exists('exception_controller', $v))
+            ->then(function ($v) {
+                if (isset($v['exception_controller'])) {
+                    throw new InvalidConfigurationException('Option "exception_controller" under "twig" must be null or unset, use "error_controller" under "framework" instead.');
+                }
+
+                unset($v['exception_controller']);
+                trigger_deprecation('symfony/twig-bundle', '7.4', 'Setting the "exception_controller" option under "twig" to null is deprecated.  Omit this legacy no-op option instead.');
```

Comment:
extra whitespace 🕵️‍♂️
```suggestion
                trigger_deprecation('symfony/twig-bundle', '7.4', 'Setting the "exception_controller" option under "twig" to null is deprecated. Omit this legacy no-op option instead.');
```